packages/kbn-optimizer/limits.yml

```diff
@@ -4,6 +4,7 @@ pageLoadAssetSize:
   aiops: 10000
   alerting: 106936
   apm: 64385
+  assetManager: 25000
```

Comment:
The bundle looks to be only 2303.  It's recommended to keep it above a level where it doesn't constantly trigger overages but below a level that when something large gets in it's flagged so an optimization can be done.  https://www.elastic.co/guide/en/kibana/current/ci-metrics.html#ci-metric-resolving-overages .  I'm thinking this could be much lower for our benefit.

Reply:
Yikes, that's some very loose guidance. I set it to 1 and ran update and I thought it was at 13000 but probably read it wrong. What exact value do you recommend? Seems like the script should set it to some function of the current value to do this calc for you.

Reply:
Yea, i'd rather be able to set a limit per PR that flags when something huge gets in.  Oftentimes reviewers/authors aren't looking at bundle increases and it gets missed.  When I had to increase [observability's limit](https://github.com/elastic/kibana/pull/159413#discussion_r1227150261) I was able to look back to a recent PR that was the culprit, adding 36KB.  Maybe 25000 is fine.  Small enough to catch any bid additions while giving us room to grow the plugin as we had queries.

Reply:
That's what I was thinking. Especially as this is a plug-in under active initial development so we expect it to grow. Once it stabilizes, we should keep it closer to the actual value probably.